nixos/modules/security/acme.nix

```diff
@@ -24,7 +24,7 @@ let
       Type = "oneshot";
       User = "acme";
       Group = mkDefault "acme";
-      UMask = 0023;
+      UMask = 0022;
```

Comment:
For context, this umask was first introduced in https://github.com/NixOS/nixpkgs/pull/91121, relaxed once in https://github.com/NixOS/nixpkgs/pull/106857 and again here.

Reply:
I would assume that acme creates sensitive files with the appropriate permissions regardless of umask. Can we confirm this in the test?

Reply:
Yes acme + lego are very delicate with the permissions they apply. I've added some tests now :)